src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -156,13 +169,14 @@ internal override void OnSelectorItemIsSelectedChanged(SelectorItem container, b
 				{
 					if (!SelectedItems.Contains(item))
 					{
-						if (!IsSelectionMultiple
-							&& SelectedItems.FirstOrDefault() is object selectedItem)
+						if (!IsSelectionMultiple && SelectedItems.Count > 0)
```

Comment:
To note, as per [UWP](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.selectionmode?view=winrt-19041#Windows_UI_Xaml_Controls_ListViewBase_SelectionMode) 
"When SelectionMode is Single, use the Selector.SelectedItem property to get the item that's selected. When SelectionMode is Multiple or Extended, use the SelectedItems property to get the items that are selected."
I *think* I've covered this in #4096 if you want to take a look.

Reply:
Cool! Thanks, I will try to replicate that here :-) . I suggest we use your changes in case of conflicts, as my changes were mainly done to cover the issues that come up for TreeView unit tests, so there might still be differences from what UWP does :-) .